about.html

```diff
@@ -106,10 +106,10 @@ <h2>
         <div class="col col-lg-6">
           <div class="description">
             <h2>
-              A system that works in the real world
+              Discover how deep the Wabbit hole goes

```

Comment:
wut?

Reply:
Haha, just taking chances. Play on words from the Matrix (and Alice in Wonderland...)